platform/android/java/lib/src/org/godotengine/godot/GodotIO.java

```diff
@@ -287,6 +294,21 @@ public int getScreenOrientation() {
 		}
 	}
 
+	public int getCurrentScreenRotation() {
+		int rotation = ((WindowManager)ContextCompat.getSystemService(activity.getBaseContext(), WindowManager.class)).getDefaultDisplay().getRotation();
+
```

Comment:
Just use `activity.getSystemService(...)` to retrieve the `WindowManager`, not need to use the `ContextCompat` class.